integration-tests/src/test/kotlin/org/stellar/anchor/platform/subtest/Sep31Tests.kt

```diff
@@ -25,32 +27,39 @@ import org.stellar.anchor.apiclient.TransactionsSeps
 import org.stellar.anchor.auth.AuthHelper
 import org.stellar.anchor.platform.*
 import org.stellar.anchor.util.GsonUtils
-import org.stellar.anchor.util.Sep1Helper.TomlContent
 import org.stellar.anchor.util.StringHelper.json
 
 lateinit var savedTxn: Sep31GetTransactionResponse
 
-class Sep31Tests(config: TestConfig, toml: TomlContent, jwt: String) {
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@Execution(SAME_THREAD)
+@TestMethodOrder(MethodOrderer.OrderAnnotation::class)
+class Sep31Tests : SepTests(TestConfig(testProfileName = "default")) {
   private val sep12Client: Sep12Client
   private val sep31Client: Sep31Client
   private val sep38Client: Sep38Client
   private val platformApiClient: PlatformApiClient
 
   init {
     println("Performing SEP31 tests...")
```

Comment:
nit: Maybe we should remove this. It doesn't seem helpful if we are running test classes concurrently. The same applies to other classes if you agree.

Reply:
These are necessary because there are relationships between test cases and they must be executed in order. This is to override the default `CONCURRENT` and `PER_METHOD` configurations.

Reply:
Sorry, I meant only removing the `println` 😅